.github/workflows/docker.yaml

```diff
@@ -29,6 +29,46 @@ jobs:
   build-and-push:
     runs-on: ubuntu-latest
     steps:
+      - id: release-params
```

Comment:
Omg.. How complex it is :D

Reply:
This could, potentially, be put into a separate GH action. I think some common functions like getting the latest release would be universally useful.